packages/cf-style-container/src/index.js

```diff
@@ -42,6 +42,7 @@ const applyTheme = (ComponentToWrap, ...themes) => {
     }
   }
 
+  ThemedComponent.displayName = `Themed${ComponentToWrap}`;
```

Comment:
WHOOPS. This was supposed to have a `.displayName`!

I went into the `node_modules` folder in the dashboard and edited the compiled source and then copy pasted it over to the `cf-ui` folder. Not sure how I missed this tho :E